monai/apps/utils.py

```diff
@@ -249,3 +251,22 @@ def download_and_extract(
     """
     download_url(url=url, filepath=filepath, hash_val=hash_val, hash_type=hash_type)
     extractall(filepath=filepath, output_dir=output_dir, hash_val=hash_val, hash_type=hash_type)
+
+
+def split_dataset(nfolds: int, length: int):
```

Comment:
it seems this still has the issue, `split_dataset(int(1 / 0.8), 5)` -> `[[0, 5]]`

Reply:
Thanks for your comment.
updated to support Kfolds cross-validation in the latest commit.
Thanks.